copycat/runtime/src/main/java/org/apache/kafka/copycat/runtime/Worker.java

```diff
@@ -106,6 +112,17 @@ public void stop() {
         long started = time.milliseconds();
         long limit = started + config.getLong(WorkerConfig.TASK_SHUTDOWN_GRACEFUL_TIMEOUT_MS_CONFIG);
 
+        for (Map.Entry<String, Connector> entry : connectors.entrySet()) {
+            Connector conn = entry.getValue();
+            log.warn("Shutting down connector {} uncleanly; herder should have shut down connectors before the" +
```

Comment:
I'm not clear why this is unclean shutdown... we are just calling conn.stop(), isn't this clean?
Also, why are we not calling stopConnector() below?